internal/store/image.go

```diff
@@ -111,78 +138,66 @@ func (b *imageStore) List() ([]reference.Reference, error) {
 	return references, nil
 }
 
-// Remove removes a bundle from the bundle store.
-func (b *imageStore) Remove(ref reference.Reference, force bool) error {
-	if id, ok := ref.(ID); ok {
-		refs := b.refsMap[id]
-		if len(refs) == 0 {
-			return fmt.Errorf("no such image %q", reference.FamiliarString(ref))
-		} else if len(refs) > 1 {
-			if force {
-				var failures *multierror.Error
-				toDelete := append([]reference.Reference{}, refs...)
-				for _, r := range toDelete {
-					if err := b.doRemove(r); err != nil {
-						failures = multierror.Append(failures, err)
-					}
-				}
-				return failures.ErrorOrNil()
-			}
-			return fmt.Errorf("unable to delete %q - App is referenced in multiple repositories", reference.FamiliarString(ref))
-		}
-		ref = refs[0]
-	}
-	return b.doRemove(ref)
-}
-
-func (b *imageStore) doRemove(ref reference.Reference) error {
-	path, err := b.storePath(ref)
+func (b *imageStore) listIDs() ([]string, error) {
+	f, err := os.Open(filepath.Join(b.path, "contents", "sha256"))
 	if err != nil {
-		return err
-	}
-	if _, err := os.Stat(path); os.IsNotExist(err) {
-		return errors.New("no such image " + reference.FamiliarString(ref))
+		return nil, err
 	}
-	b.refsMap.removeRef(ref)
-
-	if err := os.RemoveAll(path); err != nil {
-		return nil
+	defer f.Close()
+	ids, err := f.Readdirnames(-1)
```

Comment:
Does this return directories like `.` or `..` ?

Reply:
only actual directories